Lab#2/index.html

```diff
@@ -0,0 +1,82 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/base.css">
+	<link rel="stylesheet" href="css/640px.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" 
+          integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <title>Awesome Free PSD tempaltes & Resources</title>
+</head>
+<body>
+        <nav class="navigation">
+            <ul class="hidden-panel">
+                <li class="nav-element"><a class="nav-link" href="#">PSD WEBSITES</a></li>
```

Comment:
Better to use (according to BEM): 
<li class="navigation__panel-item"> 
  <a class="navigation__panel-link">

Reply:
There is no nesting from elements in BEM, as i can see in documentation. But if i have some nested elements like section->div->a->i = banner->banner__content->banner__content-link->**,  what is the template for i?

Reply:
Which BEM do you use: [getbem](http://getbem.com), [yandex bem](https://en.bem.info/methodology/) or some other? 
Because I was talking about [this rule](https://en.bem.info/methodology/html/#nesting-of-elements)

Reply:
The same rule as you, for example. The pattern is like navigation->navigation__** for infinite depth. But according to your example it is different

Reply:
But you don't have < i > element on this line, do you? I see li > a

Reply:
On this line yes. I explored the rules and some more examples, maybe for the first time it was a misunderstanding from my side. I made a try, so check the latest commit

Reply:
Let's talk about offline. For now it's okay, approved :)